prow/cluster/monitoring/README.md

```diff
@@ -68,11 +68,6 @@ Developing a new dashboard can be achieved by
 
 * Add `bazel` target to [dashboards_out/BUILD.bazel](grafana_dashboards/BUILD.bazel) for generating the configMap with the json file above.
 
-    ```
-    ### if you want to apply the configMaps
-    $ bazel run //prow/cluster/monitoring/mixins/dashboards_out:grafana-configmaps.apply
```

Comment:
@fejta this bazel target no longer exists as of https://github.com/kubernetes/test-infra/commit/1448acef54ce9c2ab4b25dcd13bc66eb69a7af46#diff-5bee9d18e9a2ba62086a15a711aba3d1

It wasn't clear if there was an alternative target I should replace this with for the docs (and for testing).